### PR TITLE
5.0 preview 3 release notes adjustments

### PR DESCRIPTION
**Pull request by David-Engel**

### CODE REVIEW

**Review by Kaur-Parminder — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,7 +39,6 @@ This update brings the below changes over the previous release:
 - Parallelize SSRP requests (instance name resolution) on Linux and macOS when MultiSubNetFailover is specified. [#1578] (https://github.com/dotnet/SqlClient/pull/1578)
 - Adjust the default ConnectRetryCount against Azure Synapse OnDemand endpoints [#1626] (https://github.com/dotnet/SqlClient/pull/1626)
 
-
 ### Changed
 
 - Dropped the `Microsoft.Data.SqlClient.Server` namespace and replaced it with supported types from the [Microsoft.SqlServer.Server](https://github.com/dotnet/SqlClient/tree/main/src/Microsoft.SqlServer.Server) package.[#1585](https://github.com/dotnet/SqlClient/pull/1585)
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:44]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 This update brings the below changes over the previous release:
 
+### Breaking changes over preview release v5.0.0-preview2
+
+- Dropped classes from the `Microsoft.Data.SqlClient.Server` namespace and replaced them with supported types from the [Microsoft.SqlServer.Server](https://github.com/dotnet/SqlClient/tree/main/src/Microsoft.SqlServer.Server) package.[#1585](https://github.com/dotnet/SqlClient/pull/1585) The affected classes and enums are:
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:13]: Does it necessary mentioning the .NET Core here?

**Review by DavoudEshtehari — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (David-Engel) [reply CHANGELOG.md:13]: Didn't we drop them for all targets?

Comment (DavoudEshtehari) [reply CHANGELOG.md:13]: Yes, but the change over previous ones is only dropping them for netcore project. We forwarded those types in netfx before.

**Pull request merged**